common/Field_Dimensions.hpp

```diff
@@ -56,6 +57,9 @@ struct Field_Dimensions {
     inline Geometry2d::Rect OurHalf() const { return _OurHalf; }
     inline Geometry2d::Rect TheirHalf() const { return _TheirHalf; }
 
+    //We play on a 2D rectangular field, hence the length "4" of the array
+    inline std::array<Geometry2d::Line, 4> FieldBorders() const { return _FieldBorders; }
```

Comment:
Yeah sounds good, go ahead and remove the inline keyword.  I think this one is the only one that definitely benefits from the ampersand.  The others might, but it's hard to tell without actually benchmarking it.  And in this case, that's way more hassle than it's worth - performance isn't that important to us here :)